fair_mango/metrics/metrics.py

```diff
@@ -433,6 +1425,73 @@ def __init__(
 
 
 class EqualisedOddsDifference:
+    """Calculate Equalised Odds Fairness Metric using "difference" to calculate
+    the disparity between the different demographic groups present in the
+    sensitive feature.
+
+    Equalised Odds calculates the "difference" in the True Positive Rate and
+    False Positive Rate in the targets to detect if there is any bias in the
+    **model**.
```

Comment:
what does it mean concretely?

Reply:
I added the description to the compared scores. It should clear things out.